src/Compilers/Core/Portable/ReferenceManager/CommonReferenceManager.Binding.cs

```diff
@@ -190,13 +216,13 @@ internal partial class CommonReferenceManager<TCompilation, TAssemblySymbol>
                             continue;
                         }
 
-                        var peReference = resolver.ResolveMissingAssembly(binding.ReferenceIdentity);
-                        if (peReference == null)
+                        var resolvedReference = resolver.ResolveMissingAssembly(requestingReference, binding.ReferenceIdentity);
```

Comment:
Since ```requestingReference``` reference is now significant, is it valid to ```continue``` in the ```if``` above based solely on ```binding.ReferenceIdentity```?